test/cp.js

```diff
@@ -797,7 +797,7 @@ test('should not attempt to copy fifos via symlinks in directories', t => {
 test('should copy fifos directly', t => {
   try {
     shell.exec(`mkfifo ${t.context.tmp}/fifo`);
-    shell.exec(`echo -n test1 > ${t.context.tmp}/fifo`, { async: true });
+    shell.exec(`printf test1 > ${t.context.tmp}/fifo`, { async: true });
```

Comment:
> Iirc, osx's echo doesn't support the -n... (Which was why I changed it initially)

`shx echo` is platform agnostic, so it runs on OS X, Win, and Linux. The behavior is guaranteed to be the same across all machines, which is what you want out of an external dependency.